src/modules/core/components/Fields/InputStatus/InputStatus.tsx

```diff
@@ -35,18 +37,20 @@ const InputStatus = ({
   error,
   status,
   statusValues,
+  touched,
 }: Props) => {
   const { formatMessage } = useIntl();
   const errorText = typeof error === 'object' ? formatMessage(error) : error;
   const statusText =
     typeof status === 'object' ? formatMessage(status, statusValues) : status;
+  const showErrorText = !isNil(touched) && touched;
```

Comment:
Why are you using isNil from lodash here?

Reply:
There are some instances of `InputStatus` where `touched` is not needed, therefore I first check that is not `undefined` or `null` to determine if `touched` is worth checking for when hiding the error message.

I actually just noticed that there's something wrong with this conditional, I'll fix it but the `!isNil(touched)` will remain the same so my explanation above still stands

Reply:
I would just do a direct check but that's a valid check too.